sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/RocksDB.scala

```diff
@@ -1005,6 +986,125 @@ class RocksDB(
   override protected def logName: String = s"${super.logName} $loggingId"
 }
 
+// uniquely identifies a Snapshot. Multiple snapshots created for same version will
+// use a different dfsFilesUUID, and hence will have different RocksDBVersionSnapshotInfo
+case class RocksDBVersionSnapshotInfo(version: Long, dfsFilesUUID: String)
```

Comment:
nit: `RocksDBVersionSnapshotId` may be better to make it clear that it is a unique identifier